app/config/test.json

```diff
@@ -3,6 +3,27 @@
     "username": "username",
     "password": "password"
   },
+  "files": {
+    "uploads": {
+      "enabled": "true",
+      "fileCount": "1",
+      "fileKey": "files",
+      "fileMaxSize": "1MB",
+      "fileMinSize": "0KB",
+      "path": "files"
+    },
+    "permanent": "localStorage",
+    "localStorage" : {
+      "path": ""
+    },
```

Comment:
test.json only needs to define values that aren't already defined in default.json - likely don't need any of these attributes highlighted.

Reply:
mostly just there to ensure that we don't use defaults or env vars for testing.  don't necessarily know what values would be in env vars, and probably don't want large values or objectstorage... who knows? but it should be known what values are in use for tests, and this is the way.